docs/platform/developing/flashing-rpi.md

```diff
@@ -209,6 +209,7 @@ The edit profile window contains the following fields:
     > [!NOTE]
     > - While creating or editing a profile, ensure that the boot images, platform images, and the RPMs (if selected) match the target device.
     > - Ensure that there are no white spaces in the boot image, platform image, and the RPM folder paths.
+    > - In case of Ubuntu, ensure that the profile name, boot image, platform image and RPM bundles fields are specified only in English language.
```

Comment:
Please change to:

```suggestion
    > - In case of Ubuntu, ensure that the profile name, boot image, platform image, and RPM bundles fields are specified only in the English language.
```

Reply:
Fixed. Thanks for the review.